Week3Tasks/Week3/src/com/week3/Bank.java

```diff
@@ -0,0 +1,13 @@
+package com.week3;
+
+public class Bank extends Account {
+    Account accountArray[]= new Account[Integer.MAX_VALUE];
+    int i=0;
+    public void addNewAccount(){
+        Account account=new Account();
+        accountArray[i]=account; i++;
+    }
+    double displayBalance(int index) {
```

Comment:
what is the index refers to ? it should has a name id instead of index.make sure to use a naming conventions varibles